src/client_side.h

```diff
@@ -394,9 +394,11 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
 #if USE_OPENSSL
     bool switchedToHttps_;
     bool parsingTlsHandshake; ///< whether we are getting/parsing TLS Hello bytes
+    uint32_t parsedBumpedRequestCount; ///< The number of bumped HTTP requests
```

Comment:
AFAICT, this new member is more accurately described as `the number of successfully parsed HTTP request headers on a bumped client connection`. An more accurate description here may help answer the "how is this counter differs from the ones we already have?" questions.

Reply:
The `successfully` part of my suggestion may be wrong -- please double check whether we are counting successful parsing events or just parsing attempts (the last of which may have failed).

Reply:
> The "successfully" is not accurate.

Agreed, but _not_ because of:

> Later we are doing more checks and the request may dropped.

The "successfully" part is wrong because the counter is incremented when parseOneRequest() returns a non-nil pointer, indicating either that the request was successfully parsed or that parsing failed. Thus, the new counter counts failure cases as well as successful ones. The adjusted description reflects that.

I am resolving this change request.